scripts/modify_files.py

```diff
@@ -0,0 +1,54 @@
+"""
+Modifying files in python
+=========================
+
+Simple text files like ``parameters.txt``
+
+.. literalinclude:: parameters.txt
+   :linenos:
+
+can easily be read (``"r"``) via
+
+"""
+with open("parameters.txt", "r") as prm_file:
+    for line in prm_file:
+        name, value = line.split("=")
+        if name == "P2":
+            print("[parameters.txt]  value of P2 is", float(value))
+
+"""
+
+Other file formats like `YAML <https://pyyaml.org/wiki/PyYAMLDocumentation>`_ or
+`JSON <https://docs.python.org/3/library/json.html>`_ are naively supported by
+python and offer a more convenient access. For example, modifying the ``parameters.yaml``
+
+.. literalinclude:: parameters.yaml
+   :linenos:
+
+is done by
+
+"""
+import yaml
```

Comment:
Maybe JSON would be more suitable for an example, since it comes included in the standard library?

Reply:
Oh, I thought it was in the standard... Not sure if I add a pip install and leave everything as is or adapt to json...